### PR TITLE
Improve command-line UX

### PR DESCRIPTION
**Pull request by drozdziak1**
This commit removes the need to verbosely type out every option and
assumes sane defaults, including logging.

rita/src/client.rs:
rita/src/exit.rs:
* Give defaults to every option
* Clean up the docopt message
* Start on the info log level by default

### CODE REVIEW

File: rita/src/exit.rs [hunk 1]
```diff
@@ -166,7 +173,14 @@ fn main() {
     // On Linux static builds we need to probe ssl certs path to be able to
     // do TLS stuff.
     openssl_probe::init_ssl_cert_env_vars();
-    env_logger::init();
+
+    // Log at info level by default
+    match env::var("RUST_LOG") {
+        Ok(_) => env_logger::init(),
+        Err(_) => env_logger::Builder::new()
```
Comment (jkilpatr) [inline rita/src/exit.rs]: no can do, start on ERR even warn fills router storage in minutes.

Comment (jkilpatr) [reply rita/src/exit.rs]: I suppose we can change the setting on the routers by updating the startup script with the package, thoughts?

Comment (drozdziak1) [reply rita/src/exit.rs]: @jkilpatr err is the default

**Comment by drozdziak1** [comment]
It may be a good idea to **hold off on merging this** until #275 is in so that touching the CLI can also involve having the new version of `docopt` work with our code.

**Pull request closed without merging**